src/components/GetImageButton.js

```diff
@@ -1,24 +1,11 @@
 import React, {Component} from 'react';
 
-const API_KEY = 'X3aNQZj7BdSiNk8EUjNBImIKplVeWrVEfeVislMQ';
-
-export default class GetImageButton extends Component{
```

Comment:
Stateless components don't need to be in a class. Rather than exporting the class, you export a function that takes a single argument (the props passed to the component).